lib/auth/grpcserver.go

```diff
@@ -2019,21 +2019,18 @@ func maybeDowngradeRole(ctx context.Context, role *types.RoleV6) (*types.RoleV6,
 	return role, nil
 }
 
-var minSupportedSSHPortForwardingVersions = map[int64]semver.Version{
-	17: {Major: 17, Minor: 1, Patch: 0},
-}
+var minSupportedSSHPortForwardingVersion = semver.Version{Major: 17, Minor: 1, Patch: 0}
 
 func maybeDowngradeRoleSSHPortForwarding(role *types.RoleV6, clientVersion *semver.Version) *types.RoleV6 {
 	sshPortForwarding := role.GetOptions().SSHPortForwarding
 	if sshPortForwarding == nil || (sshPortForwarding.Remote == nil && sshPortForwarding.Local == nil) {
 		return role
 	}
 
-	minSupportedVersion, ok := minSupportedSSHPortForwardingVersions[clientVersion.Major]
-	if ok {
-		if supported, err := utils.MinVerWithoutPreRelease(clientVersion.String(), minSupportedVersion.String()); supported || err != nil {
-			return role
-		}
+	if supported, err := utils.MinVerWithoutPreRelease(
+		clientVersion.String(),
+		minSupportedSSHPortForwardingVersion.String()); supported || err != nil {
```

Comment:
The current PR description states:

> Don’t downgrade SSH port forwarding in roles for v18.0+

However, the minSupportedSSHPortForwardingVersion is actually set to Major: 17, Minor: 1, meaning the change ensures SSH port forwarding is not downgraded starting from version 17.1.0 and upwards, not just for v18.0+.

To make the description clearer and more accurate, could you update it to:

**Don’t downgrade SSH port forwarding in roles for clients 17.1.0 and above**

Reply:
Good point, thanks :)